services/src/pro/users/postgres_userdb.rs

```diff
@@ -50,7 +50,7 @@ where
             .await?;
 
         let quota_available =
-            crate::util::config::get_config_element::<crate::pro::util::config::User>()?
+            crate::util::config::get_config_element::<crate::pro::util::config::Quota>()?
```

Comment:
I know it was like that before but, can't we store that in the context? Its an i64 so it will not really hurt. We have to think about updating things from the settings separatly.

Reply:
We could. But then do we also store the admin username and password there?
In general, the config is, in my understanding, just a typed map. So looking something up costs nothing. And it is easily extensible with new fields without having to change dozens of tests.